cves/CVE-2017-5638.yml

```diff
@@ -119,19 +129,22 @@ discovered:
     The "apache" flag can be true, false, or nil.
     If there is no evidence as to how this vulnerability was found, then you may
     leave the entries blank except for "answer". Write down where you looked in "answer".
-  answer:
-  date:
-  automated:
-  apache:
-  contest:
+  answer: |
+    Nike Zheng found this remote code execution vulnerability and reported it. He also make
+    an attack scenerio that can be found here https://github.com/tengzhangchao/Struts2_045-Poc
+  date: 2017-03-02
+  automated: false
+  apache: false
+  contest: false
 
 subsystem:
   question: |
     What subsystems was the mistake in?
     Look at the path of the source code files code that were fixed to get
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged. Examples: "clipboard", "gpu", "ssl", "speech", "renderer"
-  answer:
+  answer: |
+    Interceptor, input validation and error handling
```

Comment:
This seems like a fairly long name. Could you shorten it down to just one or two words? Maybe just "interceptor" or something else more pertinent to the vulnerability?

Reply:
noted